test_general/api/test_app.py

```diff
@@ -617,10 +617,7 @@ def test_partition_file_via_api_not_retryable_error_code(monkeypatch, mocker):
 
     assert response.status_code == 401
 
-    # Often page 2 will start processing before the page 1 exception is raised.
-    # So we can't assert called_once, but we can assert the count is less than it
-    # would have been if we used all retries.
-    assert remote_partition.call_count < 4
+    assert remote_partition.called_once()
```

Comment:
may i ask if a multiple page pdf will still fail on this `called_once`

Reply:
I fixed this by using a pdf with fewer pages! The generator makes parallel mode even faster, so the test kept failing because too many pages get processed.